packages/ui/src/components/form-input/components/form-multi-select-v2.tsx

```diff
@@ -0,0 +1,63 @@
+import { forwardRef, useRef } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+
+import { MultiSelect, type MultiSelectOption, type MultiSelectRef } from '@/components/multi-select-v2'
+import { Field } from '@components/stacked-list'
+
+interface FormMultiSelectV2PropsType
+  extends Omit<React.ComponentPropsWithoutRef<typeof MultiSelect>, 'value' | 'onChange'> {
+  name: string
+}
+
+const FormMultiSelectV2 = forwardRef<MultiSelectRef, FormMultiSelectV2PropsType>((props, ref) => {
+  const formContext = useFormContext()
+
+  const selectRef = useRef<MultiSelectRef | null>(null)
```

Comment:
do we need this ref here? I see it is preset in `FormTextInput` as well. It can be removed, not needed here.